source/edge-server/configure.txt

```diff
@@ -496,4 +421,143 @@ Upgrade Edge Server Procedure
 
       .. code-block::
 
-         make clean up
+         edgectl clean
+
+      Then, run the Edge Server again.
+
+      .. code-block::
+
+         edgectl start
+
+   .. step:: Remove Unneeded Files
+
+      After upgrading, you can remove any files that are no longer needed.
+
+      .. code-block::
+
+         edgectl prune
+
+.. _edge-server-cycle-api-key:
+
+Cycle the Edge Server API Key
+-----------------------------
+
+To cycle the Edge Server API key:
+
+.. procedure::
+
+   .. step:: Generate a New Token in Atlas
+
+      You can generate a new Edge Server token from the 
+      :ref:`edge-server-details`. Use the :guilabel:`Regenerate Token`
+      button to get a new token, and copy it. You won't be able to see this
+      token again, and must generate a new token if you do not copy it.
+
+   .. step:: Supply the New Edge Server Token
+
+      Use ``edgectl`` to be prompted for a new token from the Edge Server 
+      instance:
+
+      .. code-block:: shell
+
+         edgectl --update-token=true
+
+      This triggers ``edgectl`` to prompt you for the token. When you provide 
+      it, the Edge Server instance exchanges this token for a secret. It uses 
+      the secret to authenticate with Atlas when it syncs data.
+
+      .. important:: Required network access
+
+         To exchange the token for an API key, ``edgectl`` must have network
+         access. You cannot cycle the API key if ``edgectl`` is not online.
+
+.. _specify-edge-server-query:
+
+Specify the Edge Server's Query
+-------------------------------
+
+You can use the ``--query`` argument with the ``edgectl configure`` command
+to specify the Edge Server instance Sync query.
+
+The ``--query`` for an Edge Server instance defaults to ``"*"``, which 
+automatically syncs all data in all of the collections in your 
+:ref:`schema <schemas>`.
+
+If you would like to filter the data that syncs to your Edge Server, you have
+two options:
+
+- Pass a simple string query. For example, ``--query="truepredicate"``.
+- Pass an object containing one or more collections and queries to sync a 
+  subset of your data with Edge Server. This may resemble:
+
+  .. code-block:: json
+
+     {
+        "YOUR-COLLECTION-NAME": "YOUR-QUERY",
+        "YOUR-COLLECTION-NAME": "YOUR-QUERY"
```

Comment:
So this is a bit confusing unfortunately:

* This actually will always be the table name in the schema
* For non-Edge Server services, it is possible that the table name is different than the collection name.
* For Edge Server services, the auto generated schema when enabling the service will ensure that the table name matches the collection name.
* If a customer adds a schema on their own to any Edge Service, they can put whatever name they want.

Should we say this is the table name and then that for most cases, the table name matches the schema name but not always? What do you think?

Reply:
Good clarification - thank you! We generally avoid referring to "tables" as this is an implementation detail that customers may associate with relational data versus documents/collections. So I'll try updating this section to be clear that this is the schema title, which will typically match the collection name. I'd like to avoid getting into more detail than that, as I think our handling of things will take care of most of these details. We can always build this content out more in the future if people are getting confused.